src/components/domain/project/projectButton/ProjectEditDeleteButton.tsx

```diff
@@ -0,0 +1,24 @@
+import { Edit3, Trash2 } from 'lucide-react';
+
+interface ProjectEditDeleteButtonProps {
+  projectId: number;
+}
+
+export default function ProjectEditDeleteButton({ projectId }: ProjectEditDeleteButtonProps) {
+  const handleDeleteProject = () => {
+    alert(`${projectId}번 프로젝트 삭제 api 호출`);
```

Comment:
<img width="734" alt="image" src="https://github.com/user-attachments/assets/8ea18b70-f899-4bea-bb0f-494ef1fbd662">
요구사항정의서에 명시된 삭제알림 모달이 결과를 나타내는 것인지 잘 모르겠는데, 삭제 시 확인 알럿이 하나 뜨면 좋을 것 같습니다! 추후 api 연동하면서 추가 부탁드립니다.

```suggestion
 if (confirm(`${projectId}번 프로젝트를 삭제하시겠습니까?`)) {
 // 삭제 api 추가
     alert(`${projectId}번 프로젝트 삭제 api 호출`);
```

Reply:
네네!! 우선 기능은 생각 안했습니다! 삭제 추가하면서 메시지박스로 확인 한번 받겠슴다